src/components/MultiselectTags/MultiselectTags.vue

```diff
@@ -114,7 +135,7 @@ export default {
 		},
 		filter: {
 			type: Function,
-			default: (element, index) => index < 5,
+			default: (_element, index) => index < 5,
```

Comment:
Why rename it?

Reply:
This was a suggestion of my IDE because the variable `element` is not used. Don't know if there's a coding guideline for this but of course i can also undo it.